test/controllers/test-utils.js

```diff
@@ -54,6 +54,57 @@
       }
       return driver;
     }
+  
+    static getElementSizes(driver, locateType, locateWord) {
```

Comment:
Consider using just "selector" instead of localeType + localeWord, all cases can be resolved using just css selectors (.class, #id)